### PR TITLE
Extend documentation

### LINKED ISSUE #82: Extend documentation in readme and readthedocs
Now that we have a paper draft and some new introduction figures it would be nice to add some of this content to the readthedocs and/or the readme.

### PR DESCRIPTION
**Pull request by florian-huber**
- Update flowchart in ``readme``
- Add flowchart to ``index.rst``
- Add figure of ``spectrum.plot()`` to ``Spectrum`` docstring

### CODE REVIEW

File: readthedocs/index.rst [hunk 1]
```diff
@@ -6,14 +6,25 @@
...
    :caption: Contents:
 
    API <api/matchms.rst>
 
+Introduction
+============
+
+Matchms allows to easily build custom spectra processing pipelines and to compute spectra similarities.
+
+.. image:: paper/flowchart_matchms.png
```
Comment (sverhoeven) [inline readthedocs/index.rst]: Correct url
```suggestion
.. image:: ../paper/flowchart_matchms.png
```

File: matchms/Spectrum.py [hunk 2]
```diff
@@ -33,6 +33,14 @@ class Spectrum:
         0.7
         spectrum1
 
+    To visually inspect a spectrum run ``spectrum.plot()``
```
Comment (sverhoeven) [inline matchms/Spectrum.py]: Can you move this code to the docstring of the plot() method

File: matchms/Spectrum.py [hunk 3]
```diff
@@ -33,6 +33,14 @@ class Spectrum:
...
+
+    .. figure:: paper/spectrum-plot-example.png
```
Comment (sverhoeven) [inline matchms/Spectrum.py]: Picture does not show, can you copy it to `readthedocs/_static/` directory

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Pictures not showing in readthedocs

File: matchms/Spectrum.py (hunk 3 above)
Comment (florian-huber) [reply matchms/Spectrum.py]: Would it then be called by ``.. figure:: _static/spectrum-plot-example.png`` ?

Comment (sverhoeven) [reply matchms/Spectrum.py]: I think so

File: matchms/Spectrum.py (hunk 2 above)
Comment (florian-huber) [reply matchms/Spectrum.py]: True...Done.

File: matchms/Spectrum.py (hunk 3 above)
Comment (florian-huber) [reply matchms/Spectrum.py]: I now added both the workflow and the spectrum.plot() example figure to the ``_static`` folder... let's see if that works.

File: matchms/Spectrum.py [hunk 4]
```diff
@@ -93,7 +85,13 @@ def clone(self):
         return clone
 
     def plot(self, intensity_from=0.0, intensity_to=None, with_histogram=False, with_expfit=False):
-        """An example docstring for a method."""
+        """To visually inspect a spectrum run ``spectrum.plot()``
+
+        .. figure:: _static/spectrum-plot-example.png
```
Comment (sverhoeven) [inline matchms/Spectrum.py]: Almost Spectrum html page is in api/ dir so 
```suggestion
        .. figure:: ../_static/spectrum-plot-example.png
```

File: readthedocs/index.rst [hunk 5]
```diff
@@ -21,7 +21,7 @@ Introduction
 
 Matchms allows to easily build custom spectra processing pipelines and to compute spectra similarities.
 
-.. image:: _static/flowchart_matchms.png
+.. image:: ../_static/flowchart_matchms.png
```
Comment (sverhoeven) [inline readthedocs/index.rst]: Spectrum plot is OK, but now flowchart is kaput
```suggestion
.. image:: _static/flowchart_matchms.png
```

Comment (florian-huber) [reply readthedocs/index.rst]: Yeah... guess I fixed it kaput.

**Review by sverhoeven — APPROVED** [review]

**Comment by florian-huber** [comment]
Thanks for the quick review Stefan!
One question: how do you inspect the created readthedocs? 
I know that I can generate it locally, but can I also look at the docs that are generated during the test runs?

**Pull request merged**